lib/Doctrine/Annotations/Metadata/AnnotationTarget.php

```diff
@@ -0,0 +1,116 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Doctrine\Annotations\Metadata;
+
+use Doctrine\Annotations\Annotation\Target;
+use function array_filter;
+use function assert;
+use function implode;
+
+/**
+ * @internal
+ */
+final class AnnotationTarget
+{
+    public const TARGET_CLASS      = 1;
```

Comment:
If we can come up with other name than _class_, we could drop `TARGET_` prefix.

Reply:
`TYPE`? `KLASS`?

Reply:
Type may work, please no non-english words. 😂

Reply:
Or just `_CLASS` if starting a constant name with an underscore is valid (on my phone rn, hard to check)

Reply:
I guess we can also use something like `Target::OnClass`/`Target::ON_CLASS`, `Target::InAnnotation`/`Target::IN_ANNOTATION` etc.
Let's leave it as is for now.